MIGRATION_PROGRESS.md

```
@@ -0,0 +1,800 @@
+- マルチエージェント: `.maBanner`, `.maEyebrow`, `.layerStack`, `.layerRow`, `.lrAdk`, `.lrA2a`, `.lrMcp`, `.layerBadge`, `.lbAdk`, `.lbA2a`, `.lbMcp`, `.layerTitle`, `.layerBody`, `.layerDesc`, `.layerFile`
+- 比較グリッド: `.cmp2Grid`, `.cmp2`, `.cmp2Sub`, `.cmp2Mult`, `.cmp2Label`
+- A2A アーキテクチャ: `.a2aArch`, `.a2aArchTitle`, `.a2aRow`, `.a2aLabel`, `.a2aBox`, `.abOrch`, `.abLocal`, `.abRemote`, `.abCard`, `.abEngine`, `.a2aArrow`
+- ユースケース: `.ucGrid`, `.uc`, `.ucIcon`, `.ucTitle`, `.ucDesc`, `.ucCode`
+- 決定フロー: `.dflow`, `.dflowTitle`, `.dflowRow`, `.dnQ`, `.dnY`, `.dnN`, `.dnArr`, `.dnInd`
+- ソース: `.srcGrid`, `.srcCard`, `.srcCardNew`, `.srcSeparator`, `.srcIcon`, `.srcTitle`, `.srcUrl`, `.srcDesc`
+- ヒーロー: `.heroEyebrow`, `.heroChips`, `.heroChip`、その他: `.mono`
+
+#### page.tsx 実装ガイド（次セッション用）
+
+**Source 型**:
+```ts
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)